mmv1/templates/terraform/examples/looker_instance_psc.tf.erb

```diff
@@ -0,0 +1,17 @@
+resource "google_looker_instance" "<%= ctx[:primary_resource_id] %>" {
+  name               = "<%= ctx[:vars]["instance_name"] %>"
+  platform_edition   = "LOOKER_CORE_ENTERPRISE_ANNUAL"
+  region             = "us-central1"
+  private_ip_enabled = false
+  public_ip_enabled  = false
+  psc_enabled        = true
+  oauth_config {
+    client_id = "<%= ctx[:vars]["client_id"] %>"
+    client_secret = "<%= ctx[:vars]["client_secret"] %>"
+  }
+  psc_config {
+    allowed_vpcs: ["projects/test-project/global/networks/test"]
+    // update only
+    // service_attachments: [{local_fqdn: "www.local-fqdn.com" target_service_attachment_uri: "projects/my-project/regions/us-east1/serviceAttachments/sa"}]
```

Comment:
are these meant to be commented out? Terraform comments use "#"

But if you are trying to make an update test, that should be done in a handwritten file. I'm not sure if one already exists for looker, but you can see our documentation here: https://googlecloudplatform.github.io/magic-modules/develop/test/test/#add-an-update-test